automation/features/step-definitions/staking.steps.js

```diff
@@ -48,23 +75,321 @@ When(/^I click on use maximum button$/, () => {
     reverse: false,
     timeoutMsg: "callout was not found",
   });
-  if (browser.getByTestId("associate-more-tokens-btn").isDisplayed()) {
-    browser.getByTestId("associate-more-tokens-btn").click();
-  } else {
-    browser.getByTestId("associate-tokens-btn").click();
-  }
+  if (stakingPage.associateMoreTokensBtn.isDisplayed()) {
+    stakingPage.associateMoreTokensBtn.click();
+  } else stakingPage.associateTokensBtn.click();
```

Comment:
chore (blocking): Duplicated code. Probably would move this to a facade step in the staking page.